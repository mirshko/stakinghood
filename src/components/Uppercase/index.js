import { Text } from "rebass";
import styled from "styled-components";

const Uppercase = styled(Text)`
  text-transform: uppercase;
  color: ${props => (props.color ? props.color : "var(--brand-opaque)")};
  letter-spacing: 0.32px;
  font-weight: var(--weight-semibold);
`;

export default Uppercase;
