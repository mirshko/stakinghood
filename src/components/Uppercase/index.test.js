import React from "react";
import { shallow } from "enzyme";

import Uppercase from "./index";

describe("<Uppercase />", () => {
  it("it works", () => {
    const wrapper = shallow(<Uppercase />);
    expect(wrapper).toMatchSnapshot();
  });
});
