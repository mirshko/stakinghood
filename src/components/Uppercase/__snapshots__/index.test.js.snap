// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Uppercase /> it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-htoDjs",
        "isStatic": false,
        "rules": Array [
          "box-sizing: border-box;",
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          "
  text-transform: uppercase;
  color: ",
          [Function],
          ";
  letter-spacing: 0.32px;
  font-weight: var(--weight-semibold);
",
        ],
      },
      "displayName": "Styled(Styled(styled.div))",
      "foldedComponentIds": Array [
        "sc-bdVaJa",
        "sc-htpNat",
      ],
      "propTypes": Object {
        "fontFamily": [Function],
        "fontWeight": [Function],
        "letterSpacing": [Function],
        "lineHeight": [Function],
        "textAlign": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-htoDjs",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
